src/gui/virtualstudio.cpp

```diff
@@ -1617,6 +1623,13 @@ void VirtualStudio::completeConnection()
         connect(this, &VirtualStudio::updatedMonitorVolume, m_monitor,
                 &Monitor::volumeUpdated);
 
+        // Setup output analyzer
```

Comment:
I understand that feedback could be generated either by use of your monitor signal getting picked up by your input, or by the studio signal (a) looping back your input signal + (b) your output signal getting picked up by input. But I'm not following on why a second detector would be required. If the first case is true, wouldn't the feedback also show up on your input signal?

Reply:
I was trying to see if we could distinguish between the two forms of feedback using two separate instances of `Analyzer`. I think using only 1 also works, perhaps even a little better. In that case we'd probably want to mute both the monitor and input volume since we can't tell the two apart.

If we have only 1 instance it would need to go before the input volume plugin or after the monitor plugin - not sure which is better but I kept the one after the monitor.